### ISSUE TITLE
No visual focus indicator for CMS unlock link

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#9919]
## Background
**Still an issue in D10**

In the CMS, there is an issue found on all edit screens regarding the unlock link. There is no visible focus indicator for this link. It is styled like a button but is not actually a button, therefore it is no behaving as the other buttons do. 

The focus should be the green border that other elements have when focused on.

Image of defective area:
<img width="108" alt="Screen Shot 2022-07-27 at 12 45 17 PM" src="https://user-images.githubusercontent.com/90111970/181318284-dd6f4ec8-3c43-46bf-af90-41c7e656b769.png">

## Accessibility Standard

WCAG version 2.0 AA, [Criterion 2.4.7](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html)

## Acceptance Criteria

- [x] Check tugboat D10 to see if updated.  
- [x] Tech determination of if this is an upstream (core/module) issue
- [x] Issue fixed if possible and Accessibility review
- [ ] If upstream issue, comment with link to issue

**Comment by EWashb** [comment]
This may require an upstream patch. This is coming from drupal

**Comment by EWashb** [comment]
For UX, we may be limited due to upstream dependencies. Could be a template issue or a lack of CSS entry. It is not a button tag.

**Comment by EWashb** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/VAgov-CMS-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/9919?planning-poker) with ZenHub @ndouglas

**Comment by ndouglas** [comment]
Edmund suggests this is a 2 and can be done immediately.

**Comment by tonytaylor** [comment]
Completion targeted for EOD today.
